### PR TITLE
added Varantfile and implemented context manager

### PR DESCRIPTION
**Pull request by kiseok7**
this patch adds Varantfile for testing and devlopment.
this patch adds implementing context manager in SSHClientInteraction class.

### CODE REVIEW

**Comment by fruch** [comment]
@kiseok7 can you rebase on master, to help me test the travis integration ?

**Comment by kiseok7** [comment]
i have no experience with rebase ans pull request.
i think i do rebase my master from upstream master.
should i pull request again?

**Comment by fruch** [comment]
@kiseok7 looks like you did well

File: paramiko_expect.py [hunk 1]
```diff
@@ -65,6 +65,12 @@ def __init__(
     def __del__(self):
         self.close()
 
+    def __enter__(self):
```
Comment (fruch) [inline paramiko_expect.py]: nice touch !!

you think you can also mention this in the readme ? how to use as context ?

**Comment by fruch** [comment]
@kiseok7 also you think you can add a section on how to use the vagrant image ? (i'm not much of a vagrant user)

**Comment by kiseok7** [comment]
Thank you. I will add it on readme :)

**Pull request merged**